src/utils/config.js

```diff
@@ -20,20 +20,18 @@ import defaultConfig from '../../default_config.json';
 import { CONFIG_VARIABLES } from './constants';
 import { ValidationError } from './error';
 import { readJSONSync, writeJSONSync } from './fs';
-import logger from './logger';
```

Comment:
I think this was actually the only place the `logger` was used.
Since we use the logger from the `oclif` now, i think we are good to delete this function